mqtt/paho/pom.xml

```diff
@@ -0,0 +1,72 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.cloudevents</groupId>
+    <artifactId>cloudevents-parent</artifactId>
+    <version>2.1.0-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+  <artifactId>cloudevents-mqtt-paho</artifactId>
+  <name>CloudEvents - Paho MQTT Binding</name>
+  <packaging>jar</packaging>
+  
+  <properties>
+     <paho.version>1.2.5</paho.version>
+     <module-name>io.cloudevents.mqtt.paho</module-name>
+  </properties>
+  
+  <dependencies>
+  	  <dependency>
+  	  	  <groupId>io.cloudevents</groupId>
+  	  	  <artifactId>cloudevents-core</artifactId>
+  	  	  <version>${project.version}</version>
+  	  </dependency>
+  	  <dependency>
+	      <groupId>io.cloudevents</groupId>
+	      <artifactId>cloudevents-json-jackson</artifactId>
+	      <version>${project.version}</version>
+	  </dependency>
+
+      <!-- Paho MQTTv5 client  -->
+      <dependency>
+          <groupId>org.eclipse.paho</groupId>
+          <artifactId>org.eclipse.paho.mqttv5.client</artifactId>
+          <version>${paho.version}</version>
+      </dependency>
+      <!-- Paho MQTTv3 client -->
+	  <dependency>
+         <groupId>org.eclipse.paho</groupId>
+         <artifactId>org.eclipse.paho.client.mqttv3</artifactId>
+         <version>${paho.version}</version>
+      </dependency>
```

Comment:
Why you import both, more than just v5? I see you're using only v5

Reply:
See the `MqttMessageFactory#createReader` factory methods and the corresponding `MqttMessageFactoryTest` class

Reply:
I see what you mean. I wonder, can we somehow develop this package in a way that it doesn't hardly depend on v3/v5, so a user don't need to bring both in their classpath? E.g. we could make both dependencies as "optional" and then, depending on what you have on the classpath, the encoder is able to pick one of the two? Does this idea vaguely make sense?

Reply:
yes, this makes a lot of sense. I can try that and update the patch